.gitmodules

```diff
@@ -1,3 +1,7 @@
-[submodule "src/main/java/me/zeroeightsix/kami/util/cape-api"]
-	path = src/main/java/me/zeroeightsix/kami/util/cape-api
-	url = https://github.com/kami-blue/cape-api
+[submodule "src/main/cape-api"]
+	path = src/main/cape-api
+	url = https://github.com/kami-blue/cape-api.git
```

Comment:
consistent usage of .git, the other urls don't have it

Reply:
https://github.com/kami-blue/bot-kt/blob/48932f7ed1cc62986dbf672941f33e235444ab0b/.gitmodules#L1-L3

:thonkkill:

Reply:
okay make the other urls in client use it